### DISCUSSION TITLE
Welcome to top-issues-action Discussions!

### DISCUSSION THREAD

**Discussion by rickstaa** [rickstaa/top-issues-action#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Matrixx420** [comment]
I'm here to work hard and learn

**Comment by rickstaa** [comment]
@Matrixx420 Amazing welcome to the community! 🎉 Feel free to create an issue/PR if you have any feature requests or suggestions for improvements.

**Reply by Matrixx420** [reply]
Okay thank you I really appreciate it
